static/js/containers/LearnerPage.js

```diff
@@ -72,6 +73,20 @@ class LearnerPage extends React.Component<*, LearnerPageProps, *> {
     )(this.getFocusedDashboard());
   }
 
+  getDocumentTitle = () => {
+    const {
+      params: { username },
+      profiles,
+    } = this.props;
+
+    if (profiles[username] !== undefined) {
+      let first = R.propOr('', 'preferred_name', profiles[username]);
+      let last = R.propOr('', 'last_name', profiles[username]);
+      return `${first} ${last} | MicroMasters Profile`.trim();
```

Comment:
did you test this to see if it works? it does not, because `profiles[username]` is an object with a key called `profile` that points to the actual profile. You'll need to do this instead:

```js
R.pathOr('', ['profile', 'preferred_name'], profiles[username]);
```

maybe a more elegant approach overall would be to do this:

```js
getDocumentTitle = () => {
  const {
    params: { username },
    profiles,
  } = this.props;
  let profilePath = [username, 'profile'];

  let first = R.pathOr('', profilePath.concat('preferred_name'), profiles);
  let last = R.pathOr('', profilePath.concat('last_name'), profiles);

  return `${first} ${last} | MITx MicroMasters Profile`
    .trim()
    .replace(/^\|\s/, '');
};
```

the current solution can, if `first` and `last` end up being `""`, render the page title as `| MicroMasters Profile`.